test/geometry/scalingtest.cpp

```diff
@@ -12,7 +12,7 @@ TEST_CASE("Geometry: scaling a Point", "[geometry][operations]") {
     PointFixture pointFixture;
 
     Point scaledPoint;
-    scale(pointFixture.point, scalingPoint, scaledPoint);
+    scale(pointFixture.point, scalingPoint.x(), scalingPoint.y(), scaledPoint);
```

Comment:
I'd rather use:
`Point scaled = scale(pointFixture.point, scalingPoint);`
or 
`const Point scaled = scale(pointFixture.point, scalingPoint);`
or
`Point scaled = pointFixture.point * scalingPoint;`
or
`const Point scaled = pointFixture.point * scalingPoint;`

An even better interface would be: `auto scaled = pointFixture.point.scale(scalingPoint);`